Website/src/rendering/src/assets/css/components/Checkout/CheckoutDetails.css

```diff
@@ -167,3 +171,13 @@
     }
   }
 }
+
+.shop-checkout {
+  h1 {
+    @apply text-xl font-bold;
+  }
+
+  .breadcrumb {
+    @apply shop-container text-blue;
```

Comment:
All shop breadcrumbs should be blue according to Todor designs. Could you please move the color  (and maybe the shop-container Tailwind utility usage if really needed) to a `ShopBreadcrumb.css` file and modify the ShopBreadcrumb.tsx component with `<nav className="breadcrumb shop-breadcrumb" ...` ?